server/index.js

```diff
@@ -44,6 +44,95 @@ app.get('/api/products/:id', (req, res, next) => {
   }
 });
 
+app.get('/api/cart/', (req, res, next) => {
```

Comment:
Be sure to handle the scenario where there is no cartId. In that situation you should respond with an empty array.